FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h

```diff
@@ -47,12 +47,12 @@ NS_SWIFT_NAME(HTTPSCallable)
  * Executes this Callable HTTPS trigger asynchronously without any parameters.
  *
  * The request to the Cloud Functions backend made by this method automatically includes a
- * Firebase Instance ID token to identify the app instance. If a user is logged in with Firebase
+ * Firebase Installations token to identify the app instance. If a user is logged in with Firebase
```

Comment:
I think it's called FCM Token now: https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessaging.h#L261

And it's not part of Firebase installations